packages/asc-ui/src/components/Toggle/Toggle.tsx

```diff
@@ -104,6 +104,7 @@ const Toggle: React.FC<Props> = ({
       onBlur={handleOnBlur}
       onKeyDown={handleOnKeyDown}
       {...otherProps}
+      tabIndex={-1} // This will enable the onblur event for this div element on all browsers
```

Comment:
no side effects? also in searchbartoggle?

Reply:
Will also work over there, the negative tabindex makes it possible to focus and blur programmatically but not by keyboard 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex